features/step_definitions/collection_steps.rb

```diff
@@ -0,0 +1,10 @@
+require 'rest-client'
+require 'test-unit'
+
+Then (/^I choose existing project$/) do
+  choose_existing_project
+end
+
+Then (/^I create TESTAPI request collection$/) do
```

Comment:
Mazs ieteikums: ja tev ir soļi, kur var būt kaut kādi mainīgie (šeit TESTAPI), to var nomainīt uz (.*) pierakstu.
Aptuveni, tad kods izskatās:
Then (/^I create (.*) request collection$/) do |collection_name|
........
end